packages/ui-components/src/Tooltip/Tooltip.tsx

```diff
@@ -64,11 +64,7 @@ export const Tooltip: FunctionComponent<TooltipProps> = ({
 
   const wrappedChildren = React.Children.map(children, child => {
     return React.cloneElement(
-      typeof children === "string" ? (
-        <span>{child}</span>
-      ) : (
-        (child as ReactElement)
-      ),
+      typeof children === "string" ? <>{child}</> : (child as ReactElement),
```

Comment:
what is the reason for switching from span to empty tag; Is it just because it's more compact and no attr are added to it?

Reply:
just to avoid creation unneeded dom element